backend/common/census_cube/data/constants.py

```diff
@@ -14,7 +14,7 @@
     "EFO:0010010": "CEL-seq2",
 }
 
-CENSUS_CUBE_PINNED_SCHEMA_VERSION = "5.0.0"
+CENSUS_CUBE_PINNED_SCHEMA_VERSION = "5.1.0"
```

Comment:
why is this value hardcoded, shouldn’t it use the implied schema version from census?